api/src/main/java/org/openmrs/module/fhir2/api/search/param/PractitionerSearchParams.java

```diff
@@ -68,6 +72,7 @@ public SearchParameterMap toSearchParameterMap() {
 		        .addParameter(FhirConstants.CITY_SEARCH_HANDLER, getCity())
 		        .addParameter(FhirConstants.STATE_SEARCH_HANDLER, getState())
 		        .addParameter(FhirConstants.POSTALCODE_SEARCH_HANDLER, getPostalCode())
-		        .addParameter(FhirConstants.COUNTRY_SEARCH_HANDLER, getCountry());
+		        .addParameter(FhirConstants.COUNTRY_SEARCH_HANDLER, getCountry())
+		        .addParameter(FhirConstants.TAG_SEARCH_HANDLER, getTag());
```

Comment:
Another "I'm not quite sure what this does" addition here that it would be good to add clarification on.

Reply:
By itself, I think this is meaningless or maybe a vestige of a previous version that passed the tag down the the DAO layer? In any case, it's harmless because there's nothing in the DAO layer that does anything with this.